pkgs/by-name/fa/fac/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, makeWrapper
```

Comment:
Prefer `makeBinaryWrapper` to `makeWrapper` for a better compatibility with Darwin architectures.